docs/guides/troubleshooting.md

```diff
@@ -0,0 +1,53 @@
+# Troubleshooting
+
+## Problems with media formats
+Make sure you are using formats that video.js can play:
+* Does your browser/OS support the type of video that you are trying to play?
+* Do you have a plugin that will add support for that video format?
+* Verify that you are using the correct mime-type/content-type for your videos. This used to determine if video.js can play a video.
```

Comment:
you are specifying the correct media type for your source, whether you are using the `<source>` element or the `src()` API method.

Maybe link to http://www.iana.org/assignments/media-types/media-types.xhtml#video ?